### ISSUE TITLE
Support ISO 216 A4 size

### ISSUE THREAD

**Issue by Zekfad** [louh/control-records#17]
It would be great to have paper sizing options.

**Comment by louh** [comment]
Indeed. However, as a US federal agency they have standardized on the US Letter size and we have a hard time convincing them of the benefit in supporting their international partners.

**Comment by louh** [comment]
I've changed my mind on this, because I was looking at implementing the more realistic records style from Alan Wake II, and it turns out the in-game assets are A4-size proportions. (Probably because they're Swedish developers!)

So I think we can make an argument that for whatever reason, canonically the US Federal Bureau of Control really does use A4 paper size 😄

**Comment by louh** [comment]
@Zekfad Let me know if this works!
